src/lib/features/Card.stories.svelte

```diff
@@ -0,0 +1,38 @@
+<script>
+  import { Meta, Story } from '@storybook/addon-svelte-csf';
+  import { UserKey, Wallet } from '/src/lib/assets';
+  import Card from './Card.svelte';
+</script>
+
+<Meta title="UI Components/Features/Card" component={Card} />
+
+<!-- Primary -->
+<Story name="Card">
+  <Card></Card>
+  <Card
+    title="320px width card"
+    icon={UserKey}
+    class="w-f320 my-4"
+  />
+  <Card
+      title="Half width card"
+      icon={Wallet}
+      class="w-1/2 mb-4"
+  >
+    <p slot="content">
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididun
+    </p>
+  </Card>
+  <Card
+    title="Another color card"
+    titleColor="cream"
+    bgColor="teal"
+    color="white"
+    >
+    <div slot="content">
+      <h4 class="font-bold text-md text-navy">Slot content title</h4>
```

Comment:
```suggestion
      <h4 class="font-bold md text-navy">Slot content title</h4>
```
by using the class without the text prefix, we include the line height styling from the styleguide.